docs/hbs-partials/sassdoc-template.hbs

```diff
@@ -0,0 +1,36 @@
+{{#> page}}
+  {{#*inline "content"}}
+    <h1>{{comment.context.name}}</h1>
+    <p>{{comment.description}}</p>
+    <p>Type: {{comment.context.type}}</p>
+    {{#if comment.context.code}}
+    <div>
```

Comment:
Yeah, we can do that using custom annotations, which are pretty easy to do.
I think we'll figure out how best to present sassdoc over some time.
Let's track that with an issue.